layer/windows_writer_test.go

```diff
@@ -198,13 +198,16 @@ func testWindowsWriter(t *testing.T, when spec.G, it spec.S) {
 
 					h.AssertNil(t, lw.Close())
 
-					f.Seek(0, 0)
+					h.AssertNil(t, f.Seek(0, 0))
```

Comment:
As you can see [here](https://github.com/buildpacks/imgutil/pull/114/checks?check_run_id=2570834684) and [here](https://github.com/buildpacks/imgutil/pull/114/checks?check_run_id=2570834809), the tests are failing since `Seek` returns 2 values as described [here](https://golang.org/pkg/os/#File.Seek).

Reply:
@yaelharel thanks, what should be the other return in this case? thanks

Reply:
I don't think that there should be any `return` in this case.
`Seek` sets the offset for the next Read or Write on file and returns the new offset and an error, if any.
Since we're not using the new offset, we can just do something similar to what you did on lines 205-210:
```
_, err = f.Seek(0,0)
h.AssertNil(t, err)
```